helm/g2p-sandbox/values.yaml

```diff
@@ -568,6 +568,32 @@ ph-ee-engine:
       annotations:
         deployTime: "{{ .Values.deployTime }}"
 
+  connector:
+    enabled: true
+    image: docker.io/openmf/ph-ee-connector-bulk:latest
```

Comment:
define AWS secrets here ( ref bulk processor)

Reply:
Added aws here similar to bulk_processor